Gemfile

```diff
@@ -23,7 +23,7 @@ group :test do
   gem 'rspec-puppet-facts'
   gem 'rspec'
   gem 'puppet-blacksmith'
-  gem 'rubocop'
+  gem 'rubocop', '0.35.0'
```

Comment:
Did you not see?

https://github.com/voxpupuli/modulesync_config/pull/59#event-516658545